src/operations/command.ts

```diff
@@ -156,22 +155,3 @@ export abstract class CommandOperation<T> extends AbstractCallbackOperation<T> {
     return server.commandAsync(this.ns, cmd, options);
   }
 }
-
-/** @internal */
-export abstract class CommandCallbackOperation<T = any> extends CommandOperation<T> {
```

Comment:
Commenting here, but unrelated. We can also remove AbstractCallbackOperation as well now correct?